src/UserConfig.elm

```diff
@@ -1651,6 +1651,411 @@ Podemos cerrar su cuenta de inmediato si pensamos que ha:
     }
 
 
+thaiTexts : Texts
+thaiTexts =
+    { addEvent = "เพิ่มกิจกรรม"
+    , addressTooLong = \length maxLength -> "ที่อยู่ยาว " ++ String.fromInt length ++ " ตัวอักษร กรุณาตัดให้สั้นลงอยู่ใน " ++ String.fromInt maxLength ++ " ตัวอักษร"
+    , addressTooShort = \length minLength -> "ที่อยู่สั้น " ++ String.fromInt length ++ " ตัวอักษร ต้องมีอย่างน้อย " ++ String.fromInt minLength ++ " ตัวอักษร"
+    , aLoginEmailHasBeenSentTo = "อีเมลล็อกอินถูกส่งไปที่ "
+    , anAccountDeletionEmailHasBeenSentTo = "อีเมลล์สำหรับลบบัญชีถูกส่งไปที่ "
+    , andNanonymousNattendees =
+        \attendeeCount ->
+            if attendeeCount == 1 then
+                "และมีผู้เข้าร่วมที่ไม่ระบุชื่อ\nอีกหนึ่งคน"
+
+            else
+                "และมีผู้เข้าร่วมที่ไม่ระบุชื่อ\nอีก " ++ String.fromInt attendeeCount ++ " คน"
+    , andOneNanonymousNattendee = "และมีผู้เข้าร่วมที่ไม่ระบุชื่อ\nอีกหนึ่งคน"
+    , aPlaceToJoinGroupsOfPeopleWithSharedInterests = "สถานที่ที่คุณสามารถเข้าร่วมกับกลุ่มคนที่มีความสนใจเดียวกัน"
+    , beginsIn = "เริ่มในอีก "
+    , belowNCharactersPlease = \n -> "ต้องมีตัวอักษรน้อยกว่า " ++ String.fromInt n ++ " ตัวอักษร"
+    , buttonOnAGroupPage = "\" ปุ่มในหน้ากลุ่ม"
+    , byContinuingYouAgreeToThe = "โดยการดำเนินการต่อคุณยอมรับ "
+    , cancel = "ยกเลิก"
+    , cancelChanges = "ยกเลิกการเปลี่ยนแปลง"
+    , cancelEvent = "ยกเลิกกิจกรรม"
+    , checkYourSpamFolderIfYouDonTSeeIt = "กรุณาตรวจสอบโฟลเดอร์สแปมหากคุณไม่เห็น"
+    , chooseWhatTypeOfEventThisIs = "เลือกประเภทของกิจกรรมนี้"
+    , codeOfConduct = "ประเด็นสำคัญในการปฏิบัติ"
+    , codeOfConduct1 = "นี่คือคำแนะนำเพื่อให้คุณปฏิบัติตามกฎ \"อย่าทำตัวเป็นคนไม่ดี\":"
+    , codeOfConduct2 = "• ให้ความเคารพกับผู้คนไม่ว่าจะเป็นเรื่องเชื้อชาติ, เพศ, ตัวตนทางเพศ, สัญชาติ, รูปลักษณ์, หรือลักษณะอื่นๆ ที่เกี่ยวข้อง."
+    , codeOfConduct3 = "• ให้ความเคารพกับผู้จัดกลุ่ม พวกเขาใช้เวลาในการจัดงานและพวกเขายินดีที่จะเชิญคนที่ไม่รู้จัก เด็ดขาดอย่าทำลายความไว้วางใจของพวกเขาในคุณ!"
```

Comment:
this reads very unnatural esp. “เด็ดขาดอย่าทำลายความไว้วางใจของพวกเขาในคุณ!” part

Reply:
Key: `codeOfConduct3`
Original: Be respectful to the group organizers. They put in the time to coordinate an event and they are willing to invite strangers. Don't betray their trust in you!
Old Translation: ให้ความเคารพกับผู้จัดกลุ่ม พวกเขาใช้เวลาในการจัดงานและพวกเขายินดีที่จะเชิญคนที่ไม่รู้จัก เด็ดขาดอย่าทำลายความไว้วางใจของพวกเขาในคุณ!
Dtinth Review: This reads very unnatural especially the latter part
**New Translation**: ให้ความเคารพกับผู้จัดกลุ่ม เนื่องจากพวกเขาใช้เวลาจัดงานและยินดีที่จะเชิญชวนคนที่ไม่รู้จัก อย่าทำให้ความไว้วางใจของพวกเขาสูญเสีย!
Why: I have rephrased the translation to make it more natural and clear in Thai.